pkg/client/query/types/context.go

```diff
@@ -0,0 +1,11 @@
+package types
+
+import (
+	cosmosclient "github.com/cosmos/cosmos-sdk/client"
+)
+
+// Context is used to distinguish a cosmosclient.Context intended
```

Comment:
It's still unclear to me why we need this even after reading the comment.

#PUC?